standalone/README.md

```diff
@@ -113,50 +182,28 @@ This will provision the necessary servers for an instance of simple-server on di
 
 - Update the vault by running:
     ```bash
-    cat terraform.tfvars | ansible-vault encrypt --vault-id ../../password_file --output terraform.tfvars.vault
+    cat terraform.tfvars | ansible-vault encrypt --vault-id ../../vault_password --output terraform.tfvars.vault
```

Comment:
We might as well make this `~/.vault_password` to match [this guy](https://github.com/simpledotorg/deployment/pull/217/files#diff-31fcde2403fd89ce080b8cb97a604cb5R2)